backend/de.metas.contracts/src/main/java/de/metas/contracts/modular/workpackage/ModularContractLogHandler.java

```diff
@@ -137,13 +136,13 @@ private void handleLogs(
 					.typeId(moduleConfig.getModularContractType().getId())
```

Comment:
minor, this should be improved, but I also didn't have time in my last task to do it and I try toDo it in my task. Just wanted to write it down